libs/application/templates/parental-leave/src/fields/DraftRequiresAction/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC } from 'react'
+
+import { useLocale } from '@island.is/localization'
+
+import { FieldBaseProps, MessageFormatter } from '@island.is/application/core'
+import { Box, Button, Text, toast } from '@island.is/island-ui/core'
+import { FieldDescription } from '@island.is/shared/form-fields'
+
+import { parentalLeaveFormMessages } from '../../lib/messages'
+
+import { SUBMIT_APPLICATION } from '@island.is/application/graphql'
+import { useMutation } from '@apollo/client'
+
+function handleError(error: string, formatMessage: MessageFormatter): void {
+  toast.error(
```

Comment:
I'm wondering if it would be nice to have an util function for that. I saw it in a couple of places later below, so you won't have to redefine the toast error each time, just a thought

Reply:
Yeah good catch, refactored it to parentalLeaveUtils.ts